### PR TITLE
Do not copy codeDir when submit trial in Kubeflow and Frameworkcontroller mode

### PR DESCRIPTION
**Pull request by SparkSnail**
Fix issue https://github.com/microsoft/nni/issues/1224

### CODE REVIEW

File: src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts [hunk 1]
```diff
@@ -211,9 +215,12 @@ class FrameworkControllerTrainingService extends KubernetesTrainingService imple
                 //upload local files to azure storage
                 await AzureStorageClientUtility.uploadDirectory(
                     this.azureStorageClient, `nni/${getExperimentId()}/${trialJobId}`, this.azureStorageShare, `${trialLocalTempFolder}`);
```
Comment (QuanluZhang) [inline src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts]: what are the files under `trialLocalTempFolder`?

File: src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts [hunk 2]
```diff
@@ -211,9 +215,12 @@ class FrameworkControllerTrainingService extends KubernetesTrainingService imple
                 //upload local files to azure storage
```
Comment (QuanluZhang) [inline src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts]: local files are not clear, suggest to make it more clear, for example, -> "upload local files, including scripts for running the trial and configuration (e.g., hyperparameters) for the trial, to azure storage"

File: src/nni_manager/training_service/kubernetes/kubeflow/kubeflowTrainingService.ts [hunk 3]
```diff
@@ -212,13 +216,17 @@ class KubeflowTrainingService extends KubernetesTrainingService implements Kuber
                 throw new Error('azureStorageClient is not initialized');
             }
             try {
-                //upload local files to azure storage
+                //upload script files to azure storage
```
Comment (QuanluZhang) [inline src/nni_manager/training_service/kubernetes/kubeflow/kubeflowTrainingService.ts]: we are thinking the same thing :). the same here.

File: src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts (hunk 2 above)
Comment (SparkSnail) [reply src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts]: fixed.

File: src/nni_manager/training_service/kubernetes/kubeflow/kubeflowTrainingService.ts (hunk 3 above)
Comment (SparkSnail) [reply src/nni_manager/training_service/kubernetes/kubeflow/kubeflowTrainingService.ts]: fixed.

**Pull request merged**